background/main.js

```diff
@@ -80,8 +80,7 @@ torrentToWeb.determineFilename = function(request)
 
 torrentToWeb.notify = function(message)
 {
-    browser.notifications.create(
-        '',
+    browser.notifications.create('notif',
```

Comment:
I think I know why you did this, and I suppose that makes sense. Although I'd suggest to change it to something clearer, like `transfer_notification`. Also, please keep it on the next line.